### ISSUE TITLE
Datainspector broken with #12?

### ISSUE THREAD

**Issue by bilderbuchi** [JuliaGtk/Gtk4Makie.jl#13]
I have recently tried to follow the upgrade to Makie 0.20 (and 0.2.0 here), and in my code I don't see the Datainspector displayed anymore, which was working before.

The good thing is that https://github.com/JuliaGtk/Gtk4Makie.jl/blob/29cdbfc409bdb475e1b3aa639d2a26d2dc0e1d42/examples/interactive.jl shows the same symptoms. I run that code, activate the inspector in the hamburger menu, but no inspector bubble/overlay appears on hovering over the plot line.

This is on Windows 10, Julia 1.9.4, and GLMakie v0.9.3, Gtk4Makie v0.2.0

**Comment by jwahlstrand** [comment]
You're right, it's broken. I swear it was initially working in #12. Calling `DataInspector()` in the REPL doesn't work either, so it's not something wrong with the callback in the menu. I'll look into it when I get a chance.

**Comment by jwahlstrand** [comment]
It turned out that DataInspector doesn't work when Gtk4Makie is set as a "real" Makie backend, which was enabled by default in 0.2.0. Since this is still experimental I'll disable it by default starting in 0.2.1, which will be released ASAP (I'm getting an error when I try to register the new version right now).

**Comment by bilderbuchi** [comment]
I confirm, 0.2.1 seems to have fixed the DataInspector.
